.editorconfig

```diff
@@ -0,0 +1,15 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
```

Comment:
You should also prefer `'` single quotes as that's what we use across the board.